kedro-datasets/README.md

```diff
@@ -17,6 +17,30 @@ Welcome to `kedro_datasets`, the home of Kedro's data connectors. Here you will
 pip install kedro-datasets
 ```
 
+### Install dependencies at a group-level
+
+Data types are broken into groups e.g. `pandas`, `spark` and `pickle`. Each group has a collection of data types e.g.`pandas.CSVDataset`, `pandas.ParquetDataset` and more. You can install dependencies for an entire group of dependencies as follows:
+
+```bash
+pip install "kedro-datasets[<group>]"
+```
+
+This installs Kedro and dependencies related to the data type group. An example of this could be a workflow that depends on the data types in `pandas`. Run `pip install "kedro-datasets[pandas]"` to install Kedro and the dependencies for the data types in the [`pandas` group](https://github.com/kedro-org/kedro-plugins/tree/main/kedro-datasets/kedro_datasets/pandas).
+
+### Install dependencies at a type-level
+
+To limit installation to dependencies specific to a data type:
+
+```bash
+pip install "kedro-datasets[<group>-<dataset>]"
```

Comment:
```suggestion
pip install "kedro-datasets[<group>-<lowercase-dataset>]"
```

?

Reply:
Extra names should be normalized already; think this is unnecessary unless start getting issues.

Ref: https://peps.python.org/pep-0685/